src/lib/prepareOas.ts

```diff
@@ -54,13 +65,18 @@ export default async function prepareOas(path: string, command: 'openapi' | 'ope
 
     const fileFindingSpinner = ora({ text: 'Looking for API definitions...', ...oraOptions() }).start();
 
-    let action: 'upload' | 'reduce' | 'validate';
+    let action: 'inspect' | 'reduce' | 'upload' | 'validate';
     switch (command) {
       case 'openapi':
         action = 'upload';
         break;
+      case 'openapi:uses':
+        // With the messaging that this utility generates, referring to this action as `inspect`
+        // and not `uses` makes more sense.
+        action = 'inspect';
```

Comment:
This is in regard to this messaging:

![Screen Shot 2022-12-07 at 4 55 59 PM](https://user-images.githubusercontent.com/33762/206329927-9feeb96b-9b55-459d-8231-d6ef7110ebcf.png)

Reply:
yeaaah another reason why I wouldn't hate `inspect` as the command name.